src/app/workspace/resource/representation/archive/archive.component.html

```diff
@@ -1,18 +1,49 @@
 <div *ngIf="src && src.fileValue.fileUrl">
     <!-- in case of an error -->
-    <app-status [status]="404" [url]="src.fileValue.fileUrl" [representation]="'archive'" *ngIf="failedToLoad"></app-status>
+    <app-status [status]="404" [url]="src.fileValue.fileUrl" [representation]="'archive'" *ngIf="failedToLoad">
+    </app-status>
 
-    <button class="download" mat-button (click)="downloadArchive(src.fileValue.fileUrl)" [disabled]="failedToLoad">
+    <div class="file-representation">
+        <div class="container">
+            <div class="contents">
+                <div class="icon">
+                    <mat-icon>
+                        description
+                    </mat-icon>
+                </div>
+                <div class="file">
+                    <p>{{originalFilename}}</p>
+                </div>
+            </div>
+        </div>
+        <div class="toolbar">
+            <!-- toolbar -->
+            <div class="action horizontal bottom">
+                <!-- vertical more button with menu to downlaod and replace file -->
```

Comment:
the word "vertical" does not make sense for me in this sentence

Reply:
It refers to the three dots menu which has three vertical dots but I can see how it can be confusing with the "horizontal" class name right above it. I'll clarify the comment.